libfuse/fs.go

```diff
@@ -260,6 +281,25 @@ func (f *FS) WithContext(ctx context.Context) context.Context {
 	return ctx
 }
 
+func (f *FS) maybeStartTrace(
+	ctx context.Context, family, title string) context.Context {
+	// TODO: Add options to enable/disable tracing, or adjust
+	// trace detail.
+	tr := trace.New(family, title)
```

Comment:
It's not expensive in it's current form, because only a finite number of traces are kept around and we're not adding any events onto the trace yet (but those are also capped).

But eventually, when I add dials to turn the tracing level up, we'd want to have it off by default.